packages/boost/darwin_clang_version.patch

```diff
@@ -0,0 +1,31 @@
+From 40960b23338da0a359d6aa83585ace09ad8804d2 Mon Sep 17 00:00:00 2001
+From: Bo Anderson <mail@boanderson.me>
+Date: Sun, 29 Mar 2020 14:55:08 +0100
+Subject: [PATCH] Fix compiler version check on macOS
+
+Fixes #440.
+---
+ src/tools/darwin.jam | 5 +++--
+ 1 file changed, 3 insertions(+), 2 deletions(-)
+
```

Comment:
Can we skip this?

Reply:
Do we really need to?
My test on a mac showed no problems after I added the `level=0`.

Reply:
with level=0 it works